example-react-app/src/app/AppMenu.tsx

```diff
@@ -18,277 +18,283 @@ export const AppMenu = (props: AppMenuProps) => {
         screenId={"ExampleCustomScreen"}
         icon={<BarsOutlined />}
         caption={"screen.ExampleCustomScreen"}
-        key={"81b693e6-2a21-48df-8313-2b352df20497"}
+        key={"9b682520-27cf-4a13-9df3-923fa6a5ae87"}
       />
       <MenuItem
         screenId={"CustomFormControls"}
         icon={<BarsOutlined />}
         caption={"screen.CustomFormControls"}
-        key={"0aafe806-4cee-47a4-a9b7-c8c425dff286"}
+        key={"6930ee01-de18-4de4-a520-37ef92e06018"}
+      />
+      <MenuItem
+        screenId={"CustomCard"}
+        icon={<BarsOutlined />}
+        caption={"screen.CustomCard"}
+        key={"1143f22b-9e80-4ad4-a2f7-d29ea5502d50"}
       />
       <MenuItem
         screenId={"CustomDataDisplayComponents"}
         icon={<BarsOutlined />}
         caption={"screen.CustomDataDisplayComponents"}
-        key={"b9f97b42-42c0-414c-840f-9a59db1496f5"}
+        key={"f4ff63e2-6569-487c-9edd-35b2be01d17e"}
```

Comment:
Please revert changes to this file if it is just re-generated keys, so that it doesn't create conflicts.